lib/feature_flipper.rb

```diff
@@ -8,4 +8,8 @@ def self.send_email?
   def self.staging_email?
     Settings.reports.server.include?('stage')
   end
+
+  def self.send_edu_report_email?
+    send_email? && (Settings.api.env.blank? || Settings.api.env != 'review')
```

Comment:
what value would be true for `Settings.api.env.blank?` but not true for `Settings.api.env != 'review'`?

Reply:
Made a minor update to have a single property `Settings.api_env`. Intent is to make sure that even if the property is removed (is blank), method returns true for sending email.

Reply:
aren't all blank values going to be `!= 'review'`? for what value is `(Settings.api.env.blank? || Settings.api.env != 'review')` going to return true and `Settings.api.env != 'review'` will not return true?

Reply:
Firstly the code is outdated as the key is now updated to `api_env`. It's a multiple condition check. Consider the first call to method (`send_email`) return true and then either Settings.api_env.blank? OR Settings.api_env != 'review' returns true. For example, a scenario where the key api_env is set to no value on staging settings yml. I still want the emails being sent, unless that key value is explicitly set to `review`

Reply:
I am open to removing it if that additional check is not needed, as that condition also covers the case where the key is removed all together